Product/Production/Gateway/CONNECTGatewayWeb/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/PatientCorrelationServiceSecured.java

```diff
@@ -43,8 +41,7 @@
  *
  * @author Sai Valluripalli
  */
-@WebService
-@BindingType(value = SOAPBinding.SOAP12HTTP_BINDING)
+@BindingType(value = javax.xml.ws.soap.SOAPBinding.SOAP12HTTP_BINDING)
```

Comment:
We already have a javax.xml.ws.soap.SOAPBinding import. Why we need to define a fully qualified class name in @BindingType annotation.